parser/src/__tests__/choose.test.ts

```diff
@@ -0,0 +1,32 @@
+import { groupByUniqueName, parseMachinesFromFile } from "..";
+
+describe("Choose parsing", () => {
+  it("Should be able to grab actions and guardsdeclared inside a choose action", () => {
+    const result = parseMachinesFromFile(`
+      createMachine({
+        entry: [
+          choose([
```

Comment:
I wonder - one problematic pattern with choose is that `choose` could actually be declared within the implementations argument. Is such a situation handled?

Reply:
No - it isn't, I'll add an issue:

https://github.com/statelyai/xstate-vscode/issues/70